jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ContainerizerIntegrationTest.java

```diff
@@ -383,10 +345,7 @@ private JibContainer buildImage(
             .setLabels(ImmutableMap.of("key1", "value1", "key2", "value2"))
             .setFileEntriesLayers(fakeLayerConfigurations);
 
-    Path cacheDirectory = temporaryFolder.newFolder().toPath();
     containerizer
```

Comment:
so now we're just using the system default cache?

Reply:
Yeah, the usual place for base images. I think that's fine. We've been not setting cache directories in other integration tests.